### PR TITLE
new icon: grpc (original, plain)

### LINKED ISSUE #467: Icon request: grpc
### About the icon
rpc framework
### Links
https://grpc.io/

### PR DESCRIPTION
**Pull request by ConX**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

This PR closes #467

## Link to prove your SVG is correct and up-to-date.

https://grpc.io/img/logos/grpc-logo.png

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -7289,7 +7314,7 @@
             "programming"
         ],
         "versions": {
-            "svg": [ 
+            "svg": [
```
Comment (lunatic-fox) [inline devicon.json:7317]: Let's keep changes only to `grpc` entry.
```suggestion
            "svg": [ 
```

File: devicon.json [hunk 2]
```diff
@@ -7301,7 +7326,7 @@
             ]
         },
         "color": "#3775a9",
-        "aliases": []  
+        "aliases": []
```
Comment (lunatic-fox) [inline devicon.json:7329]: Same thing here.
```suggestion
        "aliases": []  
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Suggestions based on: https://grpc.io/img/logos/grpc-logo.svg
|`original`|`plain`|
|:-:|:-:|
|original|original|

Changes:
- `original` gradient was preserved
- `plain` based on contrast

**Comment by ConX** [comment]
I updated both based on @lunatic-fox's suggestions.

**Review by Snailedlt — COMMENTED** [review]
Looks good!
Only got an alt name suggestion, otherwise it's excellent

File: devicon.json [hunk 3]
```diff
@@ -3749,6 +3749,31 @@
...
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#5FC6C5",
```
Comment (lunatic-fox) [inline devicon.json:3774]: I forgot to suggest this. Maybe we can change the color from `#5FC6C5` to `#00b0ad`, that's the major color of its gradient.
```suggestion
        "color": "#00b0ad",
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — DISMISSED** [review]
Looks awesome. Thanks! 🚀

image
image

**Review by lunatic-fox — APPROVED** [review]
Great! 🚀
Thank you for this contribution and @Panquesito7 for the screenshots! ✔

**Pull request closed without merging**